### PR TITLE
Update PHPUnit to 9.5

### PR DESCRIPTION
**Pull request by RyanNerd**
- PhpUnit upgraded to 9.5.0
- Annotation `@doesNotPerformAssertions` added to tests that have no assertions.
- Removed `syntaxCheck` attribute from phpunit.xml as this is no longer a valid attribute.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,15 @@
 ARG PHP_VERSION=7.2
 FROM php:${PHP_VERSION}-cli
 
+# Install pickle to help manage extensions
```
Comment (duncan3dc) [inline Dockerfile:4]: Can you revert this change please, we're going to stick with pecl for now

Comment (RyanNerd) [reply Dockerfile:4]: Should I revert to what is currently in master?
image

Comment (duncan3dc) [reply Dockerfile:4]: Yes please

**Comment by RyanNerd** [comment]
I'm working grave at my regular job so I need to grab some sleep now. Hopefully you don't need me for any more changes. I'll check back in about 8 hours.

**Comment by duncan3dc** [comment]
That's great @RyanNerd thanks for your help!
